templates/_homepage.html

```diff
@@ -1,25 +1,35 @@
+{% load emojificate %}
 {% load humanize %}
 {% load i18n %}
 {% load package_tags %}
-{% load emojificate %}
 {% load typogrify_tags %}
+{% load waffle_tags %}
 
-<div class="container">
-    <div class="row">
-        {% include "homepage/django_package_of_the_week_partial.html" %}
-        {% include "homepage/grid_of_the_week_partial.html" %}
-        {% include "homepage/public_service_announcement_partial.html" %}
+
+{% switch "switch_use_grid_of_the_week" %}
+    <div class="container">
+        <div class="row">
+            {% include "homepage/grid_of_the_week_partial.html" %}
+            {% include "homepage/django_package_of_the_week_partial.html" %}
+            {% switch "switch_use_psa" %}
+                {% include "homepage/public_service_announcement_partial.html" %}
+            {% endswitch %}
+        </div>
     </div>
-</div>
+{% endswitch %}
```

Comment:
I really like this. I haven't had the opportunity to use `waffle` before, but this makes me want to try it out 😄